### ISSUE TITLE
Receiving Error: o module named behavex.__main__; 'behavex' is a package and cannot be directly executed

### ISSUE THREAD

**Issue by joshdavila** [hrcorval/behavex#31]
**Describe the bug**

After installing behavex I attempted to execute per the documented command (python -m behavex -t @Example --parallel-processes 2 --parallel-scheme scenario) and received the error "No module named behavex.__main__; 'behavex' is a package and cannot be directly executed".

I attempted to run just the behavex command (python -m behavex) and received the same error.  Please advise how I can correct this issue.

I verified the install of behavex was successful (see below):

**Note: all personal/company data has been removed

*******
Microsoft Windows [Version 10.0.19042.1706]
(c) Microsoft Corporation. All rights reserved.

C:\Program Files\Python380\Scripts>python -m pip install behavex
Defaulting to user installation because normal site-packages is not writeable
Looking in indexes: [https://*****/artifactory/api/pypi/pypi-dev/simple](https://%2A%2A%2A%2A%2A/artifactory/api/pypi/pypi-dev/simple)
Collecting behavex
  Downloading [https://******/artifactory/api/pypi/pypi-dev/packages/packages/a7/eb/3b1655d5fcc45f68897f1ddf3b384f96217de4f9e7a344ee11cf9ba9b8cb/behavex-1.5.10.tar.gz](https://%2A%2A%2A%2A%2A%2A/artifactory/api/pypi/pypi-dev/packages/packages/a7/eb/3b1655d5fcc45f68897f1ddf3b384f96217de4f9e7a344ee11cf9ba9b8cb/behavex-1.5.10.tar.gz) (467 kB)

     |████████████████████████████████| 467 kB 3.3 MB/s

  Preparing metadata (setup.py) ... done
Requirement already satisfied: behave==1.2.6 in c:\users\******\appdata\roaming\python\python38\site-packages (from behavex) (1.2.6)
Requirement already satisfied: jinja2 in c:\users\******\appdata\roaming\python\python38\site-packages (from behavex) (3.0.3)
Collecting configobj
  Downloading [https://******/artifactory/api/pypi/pypi-dev/64/61/079eb60459c44929e684fa7d9e2fdca403f67d64dd9dbac27296be2e0fab/configobj-5.0.6.tar.gz](https://%2A%2A%2A%2A%2A%2A/artifactory/api/pypi/pypi-dev/64/61/079eb60459c44929e684fa7d9e2fdca403f67d64dd9dbac27296be2e0fab/configobj-5.0.6.tar.gz) (33 kB)
  Preparing metadata (setup.py) ... done
Collecting htmlmin
  Downloading [https://******/artifactory/api/pypi/pypi-dev/b3/e7/fcd59e12169de19f0131ff2812077f964c6b960e7c09804d30a7bf2ab461/htmlmin-0.1.12.tar.gz](https://%2A%2A%2A%2A%2A%2A/artifactory/api/pypi/pypi-dev/b3/e7/fcd59e12169de19f0131ff2812077f964c6b960e7c09804d30a7bf2ab461/htmlmin-0.1.12.tar.gz) (19 kB)
  Preparing metadata (setup.py) ... done
Collecting csscompressor
  Downloading [https://*******/artifactory/api/pypi/pypi-dev/packages/packages/f1/2a/8c3ac3d8bc94e6de8d7ae270bb5bc437b210bb9d6d9e46630c98f4abd20c/csscompressor-0.9.5.tar.gz](https://%2A%2A%2A%2A%2A%2A%2A/artifactory/api/pypi/pypi-dev/packages/packages/f1/2a/8c3ac3d8bc94e6de8d7ae270bb5bc437b210bb9d6d9e46630c98f4abd20c/csscompressor-0.9.5.tar.gz) (237 kB)

     |████████████████████████████████| 237 kB 6.4 MB/s

  Preparing metadata (setup.py) ... done

Requirement already satisfied: parse-type>=0.4.2 in c:\users\******\appdata\roaming\python\python38\site-packages (from behave==1.2.6->behavex) (0.6.0)
Requirement already satisfied: six>=1.11 in c:\users\*******\appdata\roaming\python\python38\site-packages (from behave==1.2.6->behavex) (1.16.0)
Requirement already satisfied: parse>=1.8.2 in c:\users\******\appdata\roaming\python\python38\site-packages (from behave==1.2.6->behavex) (1.19.0)
Requirement already satisfied: MarkupSafe>=2.0 in c:\users\*******\appdata\roaming\python\python38\site-packages (from jinja2->behavex) (2.1.0)
Using legacy 'setup.py install' for behavex, since package 'wheel' is not installed.
Using legacy 'setup.py install' for configobj, since package 'wheel' is not installed.
Using legacy 'setup.py install' for csscompressor, since package 'wheel' is not installed.
Using legacy 'setup.py install' for htmlmin, since package 'wheel' is not installed.
Installing collected packages: htmlmin, csscompressor, configobj, behavex
    Running setup.py install for htmlmin ... done
    Running setup.py install for csscompressor ... done
    Running setup.py install for configobj ... done
    Running setup.py install for behavex ... done
Successfully installed behavex-1.5.10 configobj-5.0.6 csscompressor-0.9.5 htmlmin-0.1.12

*******

**To Reproduce**
1. Install behavex 1.5.10 (python -m pip install behavex)
2. Open python project folder in Pycharm
3. Open terminal in Pycharm and execute command (python -m behavex -t @Example --parallel-processes 2 --parallel-scheme scenario)

**Expected behavior**
Expected behave feature files and associated code to be executed

**Desktop (please complete the following information):**
- OS: Windows Version 10.0.19042.1706

**Comment by gabraham-1** [comment]
I noticed the same issue while trying to run from a Jenkins EC2 ubuntu instance. I am able to run without any issues from my mac. Not sure what I am missing.
I installed behavex via `python -m pip install -r requirements.txt`

```
Requirement already satisfied: behavex==1.5.10 in /var/lib/jenkins/.local/lib/python3.8/site-packages (from -r requirements.txt (line 7)) 
```

When i run 
`python -m behavex -t @parallel_trigger` I see the error

```
/usr/bin/python: No module named behavex.__main__; 'behavex' is a package and cannot be directly executed
```

**Comment by hrcorval** [comment]
Thanks for reporting this. Basically, when installing wrapper, it generates an executable file in the python scripts folder called "behavex". So to run the test scenarios, instead of calling the python command **python -m behavex -t @example --parallel-processes 2 --parallel-scheme scenario** (using @joshdavila example), you just need to run **behavex -t @example --parallel-processes 2 --parallel-scheme scenario**.
Please, let me know if this works. Thanks!

**Comment by gabraham-1** [comment]
Thanks for the quick response. Without the `python -m` getting error `behavex: command not found`

```
python -m pip install -r requirements.txt
behavex -t @parallel_trigger

/tmp/jenkins4360026255628661708.sh: line 34: behavex: command not found
```

let me know if i missed out anything from the instructions

**Comment by hrcorval** [comment]
It seems that the python scripts folder is not in the PATH env. variable, so when executing the behavex command, it cannot be found.

**Comment by hrcorval** [comment]
Fixed in upcoming BehaveX version 1.5.11: https://github.com/hrcorval/behavex/commit/b69359061ca6403bde9fe05e91366f289107910d

**Comment by gabraham-1** [comment]
Thank you for the fix
